qa/L0_backend_python/restart/test.sh

```diff
@@ -26,16 +26,17 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 CLIENT_LOG="./restart_client.log"
-TRITON_DIR=${TRITON_DIR:="/opt/tritonserver"}
-SERVER=${TRITON_DIR}/bin/tritonserver
-BACKEND_DIR=${TRITON_DIR}/backends
-SERVER_ARGS="--model-repository=`pwd`/models --backend-directory=${BACKEND_DIR} --log-verbose=1"
+SERVER_ARGS="--model-repository=${MODELDIR}/restart/models --backend-directory=${BACKEND_DIR} --log-verbose=1"
 SERVER_LOG="./restart_server.log"
 source ../../common/util.sh
 source ../common.sh
 
 rm -fr *.log free_memory.txt
 
+if [[ ${TEST_WINDOWS} == 1 ]]; then
+    pip install pytest tritonclient[all]
```

Comment:
Maybe we can consider moving this part to utils.sh: https://github.com/triton-inference-server/server/blob/main/qa/common/util.sh#L513-L518 to remove the duplicated code in each test.

Reply:
This is a great suggestion! Originally, I figured the dependencies might change from test to test. Turns out not really..